packages/nc-gui/components/virtual-cell/QrCode.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+import { useQRCode } from '@vueuse/integrations/useQRCode'
+
+const value = inject(CellValueInj)
+
+const qrValue = computed(() => String(value?.value))
+const qrCode = useQRCode(qrValue, {
+  width: 150,
+})
+const qrCodeLarge = useQRCode(qrValue, {
+  width: 600,
+})
+
+const modalVisible = ref<boolean>(false)
```

Comment:
`ref<boolean>` is unnecessary, just using `ref` will type it correctly with `false` as default value :)